### DISCUSSION TITLE
Convert SVG font to COLR?

### DISCUSSION THREAD

**Discussion by username1290** [fonttools/fonttools#2373] [Q&A] [answered]
Hi, 
Would appreciate any tips or guidance:

Can fonttools convert a ttf SVG color font to a COLR ttf font? 
The final output I'm aiming for are WOFF/WOFF2 webfonts in COLR format, so it works in all browsers.

Or, if using fonttools as a middle step could I convert the SVG .ttf to a different format, ie UFO, and then convert the UFO file into a COLR ttf or webfont. 

Thanks.

Edit: 
Dumb related/unrelated question: 
Tried to research this as well, but why do UFO files _don't_ support external SVGs?

**Comment by arrowtype** [comment, marked as answer]
@sogen I don’t know for sure – there may be a workflow to do this – but COLR/CPAL fonts don’t support quite the same things as SVG color fonts – in particular, I know that SVG fonts can have gradients, while COLR/CPAL fonts currently cannot.

Do you have access to the actual font sources? e.g. not just SVGs, but GlyphsApp files or UFOs? If so, the most direct route may be setting up one workflow to output SVG fonts for Adobe and a second workflow to output COLR fonts for browsers.

**Reply by username1290** [reply]
Hi, 
Yes have access to the actual source files, will check workflows that output to both SVG and COLR, thanks!
